### DISCUSSION TITLE
Fresh install on Docker. Seq.json not found

### DISCUSSION THREAD

**Discussion by clabnet** [datalust/seq-tickets#1472] [Help]
Hi, 
I have started a fresh Docker Seq container, and sent many logs. It is ok.
I would try the backup and restore of messages.  I know that is required the key to restore the backup, but the Seq.json is not found. 
Who/when is write this file ?
```
docker run -it --rm datalust/seq shell
root@8e2a308c8b04:/# seqsvr show-key
No configuration file could be found at /data/Seq.json
``` 
Thank's

**Comment by nblumhardt** [comment]
Hi @clabnet - when you run the container, you'll need to mount the storage volume, e.g.:

```
docker run -it --rm -v (Some path):/data datalust/seq shell
```

If your current container isn't mounting a local storage volume, your best option is to copy its data out to a local folder using `docker cp` (https://docs.docker.com/engine/reference/commandline/cp/), and then reconfigure the container to mount that volume to `/data`.

Let me know if this helps!

Best regards,
Nick

**Reply by clabnet** [reply]
You are right, thank's Nick.
